src/main/scala/com/fulcrumgenomics/vcf/filtration/FilterSomaticVcf.scala

```diff
@@ -37,10 +37,11 @@ import scala.collection.immutable.ListMap
 
 @clp(group=ClpGroups.VcfOrBcf, description=
   """
-    |Applies one or more filters to a VCF of somatic variants. The VCF must contain genotype information
-    |for the tumor sample.  If the VCF also contains genotypes for one or more other samples, the
-    |`--sample` option must be provided to specify the sample whose genotypes to examine and whose reads
-    |are present in the BAM file.
+    |Applies one or more filters to a VCF of somatic variants.
```

Comment:
Split off the first sentence so it pretty prints better when you do `fgbio --help`

Reply:
I think this is a @nh13-ism, where @nh13 has a habit of doing this, and now others think there is some implied functional reason for doing so.  The current version, which has a full paragraph, not a single sentence, as the first paragraph produces this output:

```
...
VCF/BCF:                              Tools for manipulating VCF, BCF, or related data.
    AssessPhasing                      Assess the accuracy of phasing for a set of variants.
    FilterSomaticVcf                   Applies one or more filters to a VCF of somatic variants.
    FixVcfPhaseSet                     Adds/fixes the phase set (PS) genotype field.
...
```

`sopt` scans the first paragraph to identify the first _sentence_ and uses that as the summary.